mkosi/backend.py

```diff
@@ -37,12 +38,16 @@
     cast,
 )
 
+from .distributions import DistributionInstaller
 from .syscall import (
     blkpg_add_partition,
     blkpg_del_partition,
     block_reread_partition_table,
 )
 
+
+
+
```

Comment:
Spurious newlines

Reply:
Ah, thought I had gotten rid of them already, they're a rebase artifact. I'll remove them.

Reply:
These are still here?

Reply:
On my side this diff too is marked as outdated and I don't see these newlines in [b8580b7](https://github.com/systemd/mkosi/pull/1217/commits/b8580b79fc15031cee82efb4e8963aed52e2cb36) anymore.